src/lib/OS/SyscallHandler.cc

```diff
@@ -417,7 +420,9 @@ void SyscallHandler::handleSyscall() {
       syscallSupported |= (op == syscalls::futex::futexop::SIMENG_FUTEX_WAIT);
       syscallSupported |=
           (op == syscalls::futex::futexop::SIMENG_FUTEX_WAKE_PRIVATE);
-      syscallSupported &= (timespecPtr == 0);
+      syscallSupported |=
+          (op == syscalls::futex::futexop::SIMENG_FUTEX_WAIT_PRIVATE);
+      // syscallSupported &= (timespecPtr == 0);
```

Comment:
commented out code?

Reply:
Should this logic be re-added (given timespecPtr is from one of the syscall args? Or should it be removed?

Reply:
Maybe add a comment above the commented out logic stating: "TODO: Investigate values of the timespecPtr and usage with current impl of futex syscall" and keep the commented out code please.

Sorry I didn't realise this was my work